adapter/authzOpa/authzOpa_test.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2017 Istio Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package authzOpa
+
+// NOTE: This test will eventually be auto-generated so that it automatically supports all CHECK and QUOTA
+//       templates known to Mixer. For now, it's manually curated.
+
+import (
+	"context"
+	"testing"
+
+	rpc "github.com/googleapis/googleapis/google/rpc"
+
+	"istio.io/mixer/adapter/authzOpa/config"
+	"istio.io/mixer/pkg/adapter/test"
+	"istio.io/mixer/template/authz"
+)
+
+var (
+	policy = `package mixerauthz
+	    policy = [
+	      {
+	        "rule": {
+	          "verbs": [
+	            "storage.buckets.get"
+	          ],
+	          "users": [
+	            "bucket-admins"
+	          ]
+	        }
+	      }
+	    ]
+	
+	    default allow = false
+	
+	    allow = true {
+	      rule = policy[_].rule
+	      input.user = rule.users[_]
+	      input.verb = rule.verbs[_]
+	    }`
+	checkMethod = "data.mixerauthz.allow"
+	cfg         = &config.Params{
+		Policy:      policy,
+		CheckMethod: checkMethod,
+	}
+)
+
+func TestAccepted(t *testing.T) {
```

Comment:
Can you combine these two test methods into a single table driven test? You can keep the setup the same, make the input to the table the `user`, `verb`, and expected output `Code`.

Reply:
Merged test cases